src/pages/community/hot-section/HotSectionView.tsx

```
@@ -2,7 +1,48 @@
+  };
+
+  useEffect(() => {
+    fetchPosts();
+  }, []);
+
+  return (
+    <div className={styles.hotSectionView}>
+      <div className={styles.hotSectionView__header}>인기글</div>
+      <div className={styles.hotSectionView__list}>
+        {postList.map((item, _) =>
+          loading ? <Loading /> : <CommunityPostRow data={item} key={item.postId} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)